build-instruction.md

```diff
@@ -660,20 +659,6 @@ sudo ldconfig
 rm 97-myriad-usbboot.rules
 ```
 
-### For Windows
-
-For Intel® Movidius™ Neural Compute Stick and Intel® Neural Compute Stick 2,
-install the Movidius™ VSC driver:
-
-1. Go to the `<DLDT_ROOT_DIR>/inference-engine/thirdparty/movidius/MovidiusDriver`
-   directory, where the `DLDT_ROOT_DIR` is the directory to which the DLDT
-   repository was cloned.
-2. Right click on the `Movidius_VSC_Device.inf` file and choose **Install** from
-   the pop-up menu.
-
-You have installed the driver for your Intel® Movidius™ Neural Compute Stick
-or Intel® Neural Compute Stick 2.
-
```

Comment:
What is the motivation? I don't remember using "VSC" driver (but it's possible I just didn't use VPU on Windows much). @AndrewBakalinIntel did we use that before? Is it just very old part of instruction?

Reply:
It's very old instruction. As far is I know, Movidius VSC driver installation is not required anymore.